lib/hanami/assets/helpers.rb

```diff
@@ -186,9 +240,25 @@ def javascript(*sources)
       #   <%= stylesheet 'application' %>
       #
       #   # <link href="https://assets.bookshelf.org/assets/application-28a6b886de2372ee3922fcaf3f78f2d8.css" type="text/css" rel="stylesheet">
-      def stylesheet(*sources)
+      def stylesheet(*sources, **options)
         _safe_tags(*sources) do |source|
-          html.link(href: _typed_asset_path(source, STYLESHEET_EXT), type: STYLESHEET_MIME_TYPE, rel: STYLESHEET_REL).to_s
+          tag_options = options.merge({
```

Comment:
@accuser Shouldn't this be the opposite? With this implementation, `href`, `type` and `rel` attributes can't be overwritten by developers.